src/components/Header/Header.stories.js

```diff
@@ -5,13 +5,7 @@ import Header from './Header.js';
 export default {
   title: 'Components/Header',
   component: Header,
-  decorators: [
-    Story => (
-      <MemoryRouter>
-        <Story />
-      </MemoryRouter>
-    ),
-  ],
+  decorators: [Story => <Story />],
```

Comment:
Same here for the import of `MemoryRouter` and I think you don't need the `decorators` for your story as you don't add anything to your story anymore. If you remove this line it should still work.